commands/command_utils.py

```diff
@@ -71,15 +99,32 @@ def get_upgrade_flavour():
     return LEAPP_UPGRADE_FLAVOUR_DEFAULT
 
 
-def _retrieve_os_release_contents(_os_release_path='/etc/os-release'):
+def _retrieve_os_release_contents(_os_release_path='/etc/os-release', strip_double_quotes=True):
     """
     Retrieve the contents of /etc/os-release
 
     :rtype: dict[str, str]
     """
+
+    def strip_quotes_from_field_value(key_val_pair):
+        return (key_val_pair[0], key_val_pair[1].strip('"'))
+
     with open(_os_release_path) as os_release_handle:
         lines = os_release_handle.readlines()
-        return dict(line.strip().split('=', 1) for line in lines if '=' in line)
+
+        os_release_contents = {}
+        for line in lines:
+            if '=' not in line:
+                continue
+
+            key_val_pair = line.strip().split('=', 1)
+
+            if strip_double_quotes:
+                key_val_pair = strip_quotes_from_field_value(key_val_pair)
+
+            os_release_contents[key_val_pair[0]] = key_val_pair[1]
```

Comment:
nitpick, but I would do rather something like this:
```suggestion
            key, val = line.strip().split('=', 1)

            if strip_double_quotes:
                val = strip_quotes_from_field_value(val)

            os_release_contents[key] = val
```

The nested `strip_quotes_from_field_value` function would need to be updated as well then. Possibly it could be also renamed to `strip_quotes`.